fire/api/model/__init__.py

```diff
@@ -130,3 +130,4 @@ def registreringtil(self):
 from .geometry import *
 from .punkttyper import *
 from .sagstyper import *
+from .observationer import *
```

Comment:
Det bliver uklart, når man importerer objekter fra disse moduler via `fire.api.model`, da de bliver en del af sidstnævntes namespace med `*`-imports. Det vil være rarere, hvis man kan se, hvad der hentes ind fra hvilket modul. Samtidig føles dette ikke presserende at rette til lige nu, hvilket nok er en typisk årsag til teknisk gæld:)

Reply:
Bliver det ikke håndteret med `__all__`-listen i `fire.api.model.observationer`? Her:
https://github.com/SDFIdk/FIRE/pull/654/commits/0e8e4d3451a703b14e802632516c8897bf6fe76f#diff-cb977ba8514e88847bb76e3d86eaf7e5afc6cc9cd70cf00c907886e92b861fefR22-R34

Reply:
Jo, men det er ikke synligt i modulet, der importerer, så man skal kigge et skridt videre under alle omstændigheder for at se, hvad der kommer med.

Men, jo, `__all__` viser mere eksplicit og i toppen af modulet, hvad der bliver importeret. Men ikke så eksplicit, som hvis man angiver det i import-linjerne.

Men det er en detalje, så det er fint, hvis det bliver, som det er nu.

Reply:
Det er jo lidt en smagssag. Her har jeg fulgt praksis fra tidligere. Jeg synes egentligt også det er en fornuftig måde at gøre det på, da man kun behøver at vedligeholde èn liste over klasse der eksporteres fra et givent modul. Man kan jo importere de samme klasser både fra `fire.api.model` og `fire.api.model.observationer`.

Reply:
Dén køber jeg. Det går op for mig, at det selvfølgelig også et valg fra vores side om, hvordan API-laget skal kunne tilgås, og dét er bare én valid måde at styre, hvad der ligger i model-laget, man tilgår som udvikler eller bruger af API'en.